js_modules/dagster-ui/packages/ui-core/src/assets/auto-materialization/SensorAutomaterializationEvaluationHistoryTable.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+
+import {useQueryRefreshAtInterval} from '../../app/QueryRefresh';
+import {InstigationTickStatus} from '../../graphql/types';
+import {useQueryPersistedState} from '../../hooks/useQueryPersistedState';
+import {useCursorPaginatedQuery} from '../../runs/useCursorPaginatedQuery';
+import {ASSET_SENSOR_TICKS_QUERY} from '../../sensors/AssetSensorTicksQuery';
+import {
+  AssetSensorTicksQuery,
+  AssetSensorTicksQueryVariables,
+} from '../../sensors/types/AssetSensorTicksQuery.types';
+import {SensorFragment} from '../../sensors/types/SensorFragment.types';
+import {RepoAddress} from '../../workspace/types';
+
+import {AutomaterializationEvaluationHistoryTable} from './AutomaterializationEvaluationHistoryTable';
+import {AssetDaemonTickFragment} from './types/AssetDaemonTicksQuery.types';
+
+const PAGE_SIZE = 15;
+
+interface Props {
+  repoAddress: RepoAddress;
+  sensor: SensorFragment;
+  setSelectedTick: (tick: AssetDaemonTickFragment | null) => void;
+  setTableView: (view: 'evaluations' | 'runs') => void;
+  setTimerange: (range?: [number, number]) => void;
+  setParentStatuses: (statuses?: InstigationTickStatus[]) => void;
+}
+
+export const SensorAutomaterializationEvaluationHistoryTable = ({
+  repoAddress,
+  sensor,
+  setSelectedTick,
+  setTableView,
+  setTimerange,
+  setParentStatuses,
+}: Props) => {
+  const [statuses, setStatuses] = useQueryPersistedState<Set<InstigationTickStatus>>({
+    queryKey: 'statuses',
+    decode: React.useCallback(({statuses}: {statuses?: string}) => {
+      return new Set<InstigationTickStatus>(
+        statuses
+          ? JSON.parse(statuses)
+          : [
+              InstigationTickStatus.STARTED,
+              InstigationTickStatus.SUCCESS,
+              InstigationTickStatus.FAILURE,
+              InstigationTickStatus.SKIPPED,
+            ],
+      );
+    }, []),
+    encode: React.useCallback((raw: Set<InstigationTickStatus>) => {
+      return {statuses: JSON.stringify(Array.from(raw))};
+    }, []),
+  });
```

Comment:
There is a decent amount of duplication with this and the `Instance` version of this table. I'll come back and tidy things up in a followup.